src/dispatcher.js

```diff
@@ -7,7 +7,9 @@ import client from './client';
 export const dispatchCommand = (message) => {
   client.message = message;
   if (message.content.startsWith(client.prefix)) {
-    const command = message.content.substring(2).split(' ')[0];
+    const command = message.content
+      .substring(client.prefix.length)
```

Comment:
the unit tests caught something 😄 

Case of if we ever set the prefix to something len 3+ need to adjust for that.

Reply:
eh yeah, I figured we never would really change it or have a need to make it longer